python/list-streams/list_streams.py

```diff
@@ -0,0 +1,51 @@
+import boto3
+import time
+import random
+import botocore.exceptions
+
+retryable_exceptions = ['ClientLimitExceededException', 'InternalFailure']
+max_requests_per_second = 20
+
+def fetchNumberOfStreams():
+    kvs_client = boto3.client(service_name='kinesisvideo',
+                              region_name='us-west-2',
```

Comment:
consts & 
 recommendation is to use https://docs.aws.amazon.com/lambda/latest/dg/security-iam.html